probe-dictionary/explore.js

```diff
@@ -431,24 +431,34 @@ function shortVersion(v) {
   return v.split(".")[0];
 }
 
-function friendlyRecordingRange(firstVersion, expiry) {
-  if (expiry == "never") {
-    return `from ${firstVersion}`;
+function friendlyRecordingRangeForHistory(history, channel, filterPrerelease) {
```

Comment:
This reads much better now! I wish this was just a little more documented, as it's still difficult to parse after not looking at this for a while.

Reply:
That's a good point, i will add some.